appveyor.yml

```
@@ -2,11 +2,7 @@ version: "{build}"
 
 environment:
   matrix:
-    - PYTHON_EXE: "C:\\Python27\\python.exe"
-    - PYTHON_EXE: "C:\\Python27-x64\\python.exe"
-    - PYTHON_EXE: "C:\\Python36\\python.exe"
     - PYTHON_EXE: "C:\\Python36-x64\\python.exe"
-    - PYTHON_EXE: "C:\\Python37\\python.exe"
     - PYTHON_EXE: "C:\\Python37-x64\\python.exe"
```

What about adding the newer versions too:
```suggestion
    - PYTHON_EXE: "C:\\Python37-x64\\python.exe"
    - PYTHON_EXE: "C:\\Python38-x64\\python.exe"
    - PYTHON_EXE: "C:\\Python39-x64\\python.exe"
```